### PR TITLE
gen jets, link to leptons, test compression,10k events

### PR DESCRIPTION
**Pull request by arizzi**
update

### CODE REVIEW

File: plugins/NanoAODOutputModule.cc [hunk 1]
```diff
@@ -150,6 +152,7 @@ NanoAODOutputModule::isFileOpen() const {
 
 void 
 NanoAODOutputModule::openFile(edm::FileBlock const&) {
+  //m_file = std::make_unique<TFile>(m_fileName.c_str(),"RECREATE","", ROOT::CompressionSettings(ROOT::kLZMA, 5));
```
Comment (gpetruc) [inline plugins/NanoAODOutputModule.cc:155]: could we keep this in, but configurable (like in the PoolOutputModule) ?

**Comment by arizzi** [comment]
yes, but perhaps we can cut and paste the code from pooloutputmodule yo
have same syntax... this was just for a quick check

Il 31 ago 2017 12:09 PM, "Giovanni Petrucciani" <notifications@github.com>
ha scritto:

**Comment by gpetruc** [comment]
yep.
you can copy paste from
https://github.com/cms-sw/cmssw/blob/master/IOPool/Output/src/PoolOutputModule.cc#L47-L48
https://github.com/cms-sw/cmssw/blob/master/IOPool/Output/src/RootOutputFile.cc#L78-L116

**Comment by arizzi** [comment]
merge this meanwhile, I'm off for few hours

**Pull request merged**